implementations/rust/ockam/ockam_command/src/relay/create.rs

```diff
@@ -79,6 +83,11 @@ impl Command for CreateCommand {
         let cmd = self.parse_args(&opts).await?;
         let at = cmd.at();
         let alias = cmd.relay_name();
+        let return_timing = if cmd.no_connection_wait {
+            ReturnTiming::Immediately
+        } else {
+            ReturnTiming::AfterConnection
+        };
```

Comment:
Should we move this as a function of `cmd`, similar to what we do in the previous two lines?